### DISCUSSION TITLE
如何解决第三方overleaf平台"STZHONGS.TTF"找不到的问题

### DISCUSSION THREAD

**Discussion by xlucn** [nju-lug/NJUThesis#229] [Q&A] [answered]
```
Package njuthesis Warning: "STZHONGS.TTF" is not found in your system. The font is normally distributed with MS Windows. I have used songti for substitution.
```

可能学校的tex平台是有这个字体的，但是我使用了一个第三方平台就没有。此时如何配置字体才能解决这个问题呢？

**Comment by atxy-blip** [comment, marked as answer]
<https://github.com/nju-lug/NJUThesis/commit/93ee219a5991e70cfd6580103e356d084983f049> 增加了手动指定华文中宋字体文件位置的功能。我们假定你把拷贝来的 `STZHONGS.TTF` 上传到了项目目录下的 `font` 文件夹：

```tex
\documentclass[type=master,font-path={font/}]{njuthesis}
%\njusetname{stzhongsfile}{foo.ttf} % 以防你的字体不叫 STZHONGS.TTF
\begin{document}
  \maketitle
\end{document}
```

开发版本的模板文件可以从 <https://github.com/nju-lug/NJUThesis/actions/runs/6451794015> 页面底部的 Artifacts 获取。

**Reply by xlucn** [reply]
感谢！我稍后去试一下

**Reply by xlucn** [reply]
我用你给的commit和tex示例试了一下，字体是我在网络上下载的。但是还是有问题，报错见下图，log文件：
[output.log](https://github.com/nju-lug/NJUThesis/files/12854774/output.log)


不知道是什么问题，平台、字体文件、tex代码还是模板？
![Screenshot_2023-10-10_14-56-17](https://github.com/nju-lug/NJUThesis/assets/12032219/20279df8-9e65-4d77-8847-ebccc091709b)

**Reply by atxy-blip** [reply]
怪我，应该用 `font-path={./font/}` 或 `font-path={font/}`

**Reply by xlucn** [reply]
好了好了，非常感谢！我也是没有经验，没发现这种问题
